src/component/plotly-dash-preview/parse.js

```diff
@@ -40,10 +40,10 @@ function parse (body, req, opts, sendToRenderer) {
     result.browserSize = cst.sizeMapping[result.pdfOptions.pageSize]
   } else if (body.pageSize && isPositiveNumeric(body.pageSize.width) &&
              isPositiveNumeric(body.pageSize.height)) {
-    result.browserSize = {
-      width: body.pageSize.width * cst.pixelsInMicron,
-      height: body.pageSize.height * cst.pixelsInMicron
-    }
+    result.pdfOptions.pageSize = {
+      width: body.pageSize.width, 
+      height: body.pageSize.height
```

Comment:
This is a breaking change :\

Reply:
@tarzzz I don't think we should make such a big change to the units for `widht` and `height`. Is it not possible to update your client instead of Orca?